gsbparse/account_section.py

```diff
@@ -388,7 +388,7 @@ def _name_col(self):
 
     @property
     def _int_cols(self):
-        return ["Current_number", "Account"]
+        return ["Account"]
```

Comment:
IMO, we'd prefer to cast the value, but — as you've noticed it's not the case already — handle edge cases gracefully.
I think simply using the [`error: "coerce"` option](https://pandas.pydata.org/docs/reference/api/pandas.to_numeric.html) in the `pd.to_numeric` casting [in L83 above](https://github.com/EBoisseauSierra/gsbparse/blob/master/gsbparse/account_section.py#L83) should do (convert any non-castable input into `np.NaN`) — similar to what we do for `_date_cols`.
Admittedly, we're silencing potential errors, but the tradeoff seems acceptable.

Reply:
P.S.: another (better?) approach would be to _not_ use the `error="coerce"` option but explicitly casting `"(null)"` to `np.NaN` beforehand, so we get best of both worlds.
I'm happy with either implementation though.

Reply:
This is obviously a better way (I'm not used to pandas), I switched to `errors="coerce"`.